pkg/rancher-components/src/components/Form/Checkbox/Checkbox.vue

```diff
@@ -319,12 +319,13 @@ $fontColor: var(--input-label);
   }
 
  .checkbox-custom {
-    height: 14px;
-    width: 14px;
+    height: 14px !important;
```

Comment:
Do we have to use `!important` ? This is generally bad practice

Reply:
I caught a breakpoint (shrink window) where the box was 13x13px somehow... I just enforced it by doing the `!important` so that it never breaks it's form.

Reply:
Can't repro it... Spent 10min scaling the window again and again... Removed the `!important`